AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/SQLStatementTests.swift

```diff
@@ -836,4 +877,18 @@ class SQLStatementTests: XCTestCase {
         XCTAssertEqual(variables[4] as? String, "%gelato%")
         XCTAssertEqual(variables[5] as? String, "ice cream%")
     }
+
+    func testTranslateComplexGroupedQueryPredicateWithNamespace2() {
+
+        let predicate = field("commentPostId") == "postID"
+
+        let statement = ConditionStatement(modelSchema: Post.schema, predicate: predicate, namespace: "root")
+
+        XCTAssertEqual("""
+          and "root"."commentPostId" = ?
+        """, statement.stringValue)
+
+        let variables = statement.variables
+        XCTAssertEqual(variables[0] as? String, "postID")
+    }
```

Comment:
Unit tests should be written in the following way.  There are always exceptions to the rule, but in general, we follow the following pattern:

```
func testDescriptiveTestName() {
    // Arrange
    Instantiate variables, etc.

    // Act
    Call the code to execute the test

    // Assert
    XCTAssert(.....)
    XCTAssert(.....)
    XCTAssert(.....)
}
```
1.  Notice that there here is a single whitespace between each section.
2.  Please update the function name to describe what we are testing.  The test name is very similar to the test above, but we seem to be testing something different in this test.

Reply:
1. Ya, I agree this pattern is more clear, I was just following the pattern of other tests in this file.
2. I will change the function name

Reply:
Regarding bullet 1. Took another thought
```
func testDescriptiveTestName() {
    // Arrange
    Instantiate variables, etc.

    // Act
    Call the code to execute the test

    // Assert
    XCTAssert(.....)

    // Act
    Call the code to execute the test

    // Assert
    XCTAssert(.....)
}
```
Just my idea, It is clearer to show what action we are asserting for
eg. First, `ConditionStatement` is called and `stringValue` is asserted, Secondly, `let variables = statement.variables` and assert for `variables` alone